pallets/frequency-tx-payment/README.md

```diff
@@ -0,0 +1,54 @@
+# Frequency Transaction Pallet
+
+Allows users to perform transactions using Capacity.
+
+## Summary
+
+Frequency supports the following alternative payments:
+
+- Capacity: A refillable resource limited to a subset of transactions.
+
+The Frequency Transaction Pallet proxies or nests one or more calls inside of the root call to enable them to be paid using an alternative method.
+For example, to call something like `add_ipfs_message(params)` with Capacity, one would call `pay_with_capacity(add_ipfs_message(params))`.
+The `pay_with_capacity` will verify that the inner call is allowed with capacity.
+
+### Requirements for Paying with Capacity
+
+The account must:
+1. Be a current control key on a Provider.
```

Comment:
We might need to add that the caller (provider) should have enough capacity left on that Era to cover for that call.